setup.py

```diff
@@ -7,20 +7,23 @@
 # To use a consistent encoding
 from codecs import open
 from os import path
+import sys
 
 here = path.abspath(path.dirname(__file__))
 
 # Get the long description from the relevant file
 with open(path.join(here, 'DESCRIPTION.rst'), encoding='utf-8') as f:
     long_description = f.read()
 
+test_requires = ['coverage', 'mock>=2.0.0']
+
 setup(
     name='datasketch',
 
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='1.2.5',
+    version='1.2.7',
```

Comment:
Can you sync this branch with the current master branch, which is at version 1.2.6. You may need to resolve some mini conflicts caused by #57

Reply:
Ignore this I realized I can do this on GitHub.